c2corg_ui/views/index.py

```diff
@@ -57,6 +58,28 @@ def following(self):
         return self._get_page(
             'following', 'c2corg_ui:templates/following.html')
 
+    @view_config(route_name='whatsnewdata', renderer='json')
+    def whatsnewdata(self):
```

Comment:
Do you need this view? The request could be made directly to the API (instead of making a request to the UI server-side which makes a request to the API).